ubx/UBX/RXM.py

```diff
@@ -0,0 +1,133 @@
+"""Receiver Manager Messages: i.e. Satellite Status, RTC Status."""
+
+from ubx.UBXMessage import initMessageClass, addGet
+from ubx.Types import CH, U, U1, U4, I4, X4
+
+@initMessageClass
+class RXM:
+    """Message class RXM."""
+
+    _class = 0x02
+
+    class IMES:
+        """Indoor Messaging System information"""
+
+        _id = 0x61
+
+        class Fields:
+            pass
+
+    class IMES_GET:
+
+        _id = 0x61
+
+        class Fields:
+            numTx = U1(1)
+            version = U1(2)
```

Comment:
Example for fields with predefined values 
<img width="129" alt="image" src="https://github.com/ACRIOS-Systems/pyUBX/assets/13130700/d493dc3a-cb2b-482b-8235-ba5de1325e12">